src/Components/Nav/Nav.js

```diff
@@ -1,26 +1,79 @@
 import React, { Component } from 'react';
 import './Nav.scss';
 import { Link } from 'react-router-dom';
+import Icon from './Icon';
+import Aside from '../Aside/Aside';
+import PageTest from './PageTest';
 
 export default class Nav extends Component {
+  state = {
+    iconList: [],
+    isVisible: false,
+    isExpand: false,
+  };
+
+  expandBar = () => {
+    this.setState({
+      isExpand: !this.state.isExpand,
+    });
+  };
+
+  toggleSideBar = () => {
+    this.state.isVisible === true
+      ? this.setState({
+          isVisible: !this.state.isVisible,
+          isExpand: false,
+        })
+      : this.setState({
+          isVisible: !this.state.isVisible,
+          isExpand: false,
+        });
+  };
```

Comment:
```suggestion
  toggleSideBar = () => {
    this.setState({
      isVisible: !this.state.isVisible,
      isExpand: false,
    })
  };
```
삼항 연산자를 쓸 필요가 없는 케이스 같아요